src/hooks/shapediver/useAppBuilderSettings.ts

```diff
@@ -79,33 +79,49 @@ export default function useAppBuilderSettings(defaultSession?: IAppBuilderSettin
 		return validate(await response.json());
 	}, [url]);
 
-	// register theme overrides
-	const setThemeOverride = useThemeOverrideStore(state => state.setThemeOverride);
-	console.debug("Theme overrides", value);
-	setThemeOverride(value?.themeOverrides);
-
 	// check for ticket, modelViewUrl, slug and platformUrl
 	const ticket = parameters.get("ticket");
 	const modelViewUrl = parameters.get("modelViewUrl");
 	const slug = parameters.get("slug");
 	const platformUrl = parameters.get("platformUrl");
 	const disableFallbackUi = isTrueish(parameters.get("disableFallbackUi"));
-
+	const template = parameters.get("template");
```

Comment:
@mathoebinger the template can also be selected using this query string parameter, but only if the `g` query parameter is not used